packages/serverless-components/nextjs-component/src/component.ts

```diff
@@ -218,6 +218,11 @@ class NextjsComponent extends Component {
         ? nextConfigPath
         : resolve(inputs.build.baseDir);
 
+    const outputFileTracing =
+      typeof inputs.build !== "boolean" &&
+      typeof inputs.build !== "undefined" &&
+      !!inputs.build.outputFileTracing;
```

Comment:
probably can cover it in custom inputs test? or if it's not too convenient, we could modify the existing e2e tests to cover this

Reply:
I added a custom inputs test and also updated the `next-app-experimental` to use this flag.